extraction/src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipelineBuilder.scala

```diff
@@ -67,7 +67,7 @@ class ExtractionPipelineBuilder(getClient: () => EncodeClient)
 
     // biosamples are the first one and follow a different pattern, so we don't use the generic method
     val biosamples = ctx
-      .parallelize(List(List("organism.name" -> "human")))
+      .parallelize(List({ args.initialQuery }))
```

Comment:
Brackets not needed here
```suggestion
      .parallelize(List(args.initialQuery))
```